plugins/GSdx/GSTextureCache.cpp

```diff
@@ -2046,30 +2045,39 @@ void GSTextureCache::SourceMap::RemoveAt(Source* s)
 	delete s;
 }
 
-// Query the PaletteMap for a valid Palette, then assign both palette texture pointer and clut copy pointer to the Source object
-void GSTextureCache::AttachPaletteToSource(Source* s, uint16 pal)
+// Query the PaletteMap for a valid Palette, then both its reference, clut copy pointer and eventually palette texture pointer to the Source object
```

Comment:
So I'm having trouble parsing the comment. Did you mean to drop the assign? And what does "eventually palette texture pointer" mean?

Reply:
The "assign" was removed by error in fact, now the statements lacks a verb, sorry for that!
The "palette texture pointer" is the pointer GSTexture* (m_tex_palette in Palette) to the (in the 8-bit texture enabled case) allocated texture with as content the copy of the clut in the relevant Palette object.
This texture pointer is then assigned in the 8-bit texture enabled case to the Source object, whereas in the other case it is null and not assigned to the Source.

Reply:
Yes I misused the term "eventually" (false friend!); What I meant was "optionally", with the positive case being when the `need_gs_texture` parameter is true, and viceversa.

Reply:
So what should be the sentence changed to ? I can make a quick push.

Reply:
From the code and what the original comment was, I'd suggest:
```suggestion
// Query the PaletteMap for a valid Palette, then assign its reference, CLUT copy pointer and optionally its palette texture pointer to the Source object.
```

Reply:
The proposed change looks fine